apps/emqx_management/src/emqx_mgmt_data_backup.erl

```diff
@@ -668,18 +651,41 @@ flag_to_boolean(<<"off">>) -> false;
 flag_to_boolean(Other) -> Other.
 -endif.
 
-read_global_auth_type(Data, Version) when Version =:= "4.0" orelse
-                                          Version =:= "4.1" orelse
-                                          Version =:= "4.2" ->
+is_version_supported("4.1") ->
```

Comment:
Perhaps:
```
is_version_supported(Version) ->
  try lists:map(fun erlang:list_to_integer/1, string:tokens(Version, ".")) of 
    [4, 1|_] -> true;
    [4, 3|_] -> true;
    [4, 2, N|_] -> N >= 11;
    [4, 0, N|_] -> N >= 13
    _ -> false
  catch 
    _:_ -> false
  end
```

Reply:
I just match:
```
[4, 2, N] -> N >= 11;
[4, 0, N] -> N >= 13
```
Other version should be invalid.